jsx/StaticDataTable.js

```diff
@@ -432,7 +440,7 @@ class StaticDataTable extends Component {
 
         if (this.hasFilterKeyword(this.props.Headers[j], data)) {
           filterMatchCount++;
-          filteredData.push(this.props.Data[index[i].RowIdx]);
+          this.props.FilteredData.push(this.props.Data[index[i].RowIdx]);
```

Comment:
Props should be read-only. Components should not be altering their own props.

Reply:
@HenriRabalais The filter works fine, and the query received the correct params (AND cohort.SubprojectID LIKE CONCAT('%', :v_cohortsubprojectid, '%')) without changes needed.